src/plugins/crypto/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+find_package (LibGcrypt)
+
+set (SOURCES 
+	gcrypt_operations.h
+	gcrypt_operations.c
+	crypto.h
+	crypto.c
+)
```

Comment:
CMake file in general looks nice. SOURCES is not really needed and its a bit confusing why its outside of LIBGCRYPT_FOUND